ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -55,8 +55,7 @@ export default class SignatureRequestOriginal extends Component {
     this._removeBeforeUnload();
   };
 
-  _beforeUnload = (event) => {
-    const { clearConfirmTransaction, cancel } = this.props;
+  _beforeUnload = (_) => {
```

Comment:
Nit: This argument can be omitted
```suggestion
  _beforeUnload = () => {
```